lib/agent0/agent0/hyperdrive/policies/lpandarb.py

```diff
@@ -50,7 +50,10 @@ def calc_shares_needed_for_bonds(
         _shares_to_gov: FixedPoint
             The associated shares going to governance.
     """
-    _shares_to_pool = interface.calc_shares_out_given_bonds_in_down(abs(bonds_needed), pool_state)
+    try:
```

Comment:
why are you adding this? or, what are you trying to sovle here? The difference is rounding, yes? This seems like a bad idea to me...

Reply:
I'm going to test whether I can get away with not using it

Reply:
ran a sweep without this try/catch, so assuming it works for now. will add back later with more detail, if needed.